examples/tpm-sign/tpm-sign.go

```diff
@@ -0,0 +1,148 @@
+// Copyright (c) 2014, Google Inc. All rights reserved.
```

Comment:
Generally, the style here would be copyright Whoever with the year 2018, followed by the Apache license, IIUC.